core-ui/src/components/Extensibility/components/Panel.js

```diff
@@ -18,26 +18,47 @@ export function Panel({
     'fd-margin--md': !disableMargin,
   });
 
+  const bodyClassNames = classNames({
+    'no-padding': structure?.disablePadding,
+  });
+
+  const header = structure?.header || [];
+
   return (
     <LayoutPanel className={panelClassNames}>
       <LayoutPanel.Header>
-        <LayoutPanel.Head title={widgetT(structure)} />
-      </LayoutPanel.Header>
-      <LayoutPanel.Body>
-        {Array.isArray(structure?.children)
-          ? structure.children?.map((def, idx) => (
+        <LayoutPanel.Head
+          title={widgetT(structure)}
+          className="fd-margin-end--sm"
+        />
+        {Array.isArray(header)
+          ? header.map((def, idx) => (
               <Widget
                 key={idx}
                 value={value}
                 structure={def}
                 schema={schema}
-                inlineRenderer={InlineWidget}
                 inlineContext={true}
                 {...props}
               />
             ))
           : null}
-      </LayoutPanel.Body>
+      </LayoutPanel.Header>
+      {Array.isArray(structure?.children) ? (
+        <LayoutPanel.Body className={bodyClassNames}>
+          {structure.children?.map((def, idx) => (
+            <Widget
+              key={idx}
+              value={value}
+              structure={def}
+              schema={schema}
+              inlineRenderer={InlineWidget}
+              inlineContext={true}
+              {...props}
+            />
+          ))}
+        </LayoutPanel.Body>
+      ) : null}
```

Comment:
Let's simplify this a bit.
```suggestion
      {Array.isArray(structure?.children) && (
        <LayoutPanel.Body className={bodyClassNames}>
          {structure.children.map((def, idx) => (
            <Widget
              key={idx}
              value={value}
              structure={def}
              schema={schema}
              inlineRenderer={InlineWidget}
              inlineContext={true}
              {...props}
            />
          ))}
        </LayoutPanel.Body>
      )}
```